proof_of_stake/src/lib.rs

```diff
@@ -716,6 +721,8 @@ pub fn is_validator<S>(
 where
     S: StorageRead,
 {
+    // TODO: should this check be made different? I suppose it does work but
```

Comment:
What do you mean?

Reply:
Technically this works because all validators need to have a `max_commission_rate_change` set in storage, but it just seems odd from a readability perspective. May change this in a small PR before the next release